openphish v2 - added url parameter (#11214)
demisto/content: Packs/OpenPhish/ReleaseNotes/2_1_0.md

#### Integrations
##### OpenPhish v2

> - added url parameter in instance's configuration which enables you to define the url for the API.

Reviewer: ```suggestion
- added url parameter in instance's configuration which enables you to define the url for the API.
- Updated the Docker image to: *demisto/python3:3.9.1.15759*.
```

> - added url parameter in instance's configuration which enables you to define the url for the API.

Reviewer: ```suggestion
- added url parameter in instance's configuration which enables you to define the url for the API.
- Updated the Docker image to: *demisto/python3:3.9.1.15759*.
```

> - added url parameter in instance's configuration which enables you to define the url for the API.

Reviewer: ```suggestion
- Added the *Use secured protocol* parameter in the instance configuration which will uses HTTPS.
```
